layer/json/VkLayer_gfxreconstruct.json.in

```diff
@@ -374,31 +376,33 @@
                             "description": "When set, log messages will be written to a file at the specified path. Default is: Empty string (file logging disabled).",
                             "type": "SAVE_FILE",
                             "filter": "*.txt",
-                            "default": ""
-                        },
-                        {
-                            "key": "log_file_flush_after_write",
-                            "env": "GFXRECON_LOG_FILE_FLUSH_AFTER_WRITE",
-                            "label": "Log File Flush After Write",
-                            "description": "Flush the log file to disk after each write when true.",
-                            "type": "BOOL",
-                            "default": false
-                        },
-                        {
-                            "key": "log_file_keep_open",
-                            "env": "GFXRECON_LOG_FILE_KEEP_OPEN",
-                            "label": "Log File Keep Open",
-                            "description": "Keep the log file open between log messages when true, or close and reopen the log file for each message when false.",
-                            "type": "BOOL",
-                            "default": true
-                        },
-                        {
-                            "key": "log_file_create_new",
-                            "env": "GFXRECON_LOG_FILE_CREATE_NEW",
-                            "label": "Log File Overwrite",
-                            "description": "Specifies that log file initialization should overwrite an existing file when true, or append to an existing file when false.",
-                            "type": "BOOL",
-                            "default": true
+                            "default": "",
+                            "settings": [
```

Comment:
Question for @christophe-lunarg: I nested the the log_file_* settings under log_file because I saw a similar structure with page_guard, does this make sense?

Reply:
Yes totally!